apps/design/backend/src/app.results.test.ts

```diff
@@ -631,26 +667,98 @@ test('quick results reporting works for polls open reporting', async () => {
     })
   );
 
-  const pollsStatusUpdated = await workspace.store.getPollsStatusForElection(
-    await workspace.store.getElection(sampleElectionDefinition.election.id),
-    true
+  const pollsStatusUpdated = await apiClient.getReportedPollsStatus({
+    electionId: sampleElectionDefinition.election.id,
+  });
+  expect(pollsStatusUpdated).toEqual(
+    ok({
+      election: expect.objectContaining({
+        id: sampleElectionDefinition.election.id,
+      }),
+      isLive: false,
+      ballotHash: sampleElectionDefinition.ballotHash,
+      reportsByPrecinct: expect.objectContaining({
+        [precinctId]: [
+          {
+            machineId: 'mock-02',
+            pollsState: 'polls_open',
+            precinctSelection: singlePrecinctSelectionFor(
+              sampleElectionDefinition.election.precincts[0].id
+            ),
+            signedTimestamp: new Date('2024-05-04T09:00:00Z'),
+          },
+        ],
+        '': [
```

Comment:
I don't really love having sentinel values like this that don't really explain themselves. Perhaps make a constant?

```ts
export const ALL_PRECINCTS_REPORT_KEY = '';
```